app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html

```diff
@@ -27,4 +27,4 @@
     }
 }
 
-<h1 @{headingClasses}>@{params.heading}</h1>
\ No newline at end of file
+<h1 class="govuk-heading-l" @{headingClasses}>@{params.heading}</h1>
```

Comment:
Won't this potentially add two `class` attributes?

Reply:
could you please help me understand why this page is needed?  Isn't the page heading covered within the play ui components and upgrade?

Reply:
@adamliptrot-oc  o yes, I think we shouldn't even use this Pageheading anymore

Reply:
realised not even using it, so reverted back